templates/applications/application.html

```diff
@@ -4,136 +4,319 @@
+        </li>
+      </ol>
+    </div>
+  </section>
+{% if "stage_progress" in application and not application["stage_progress"]["assessment"]  %}
+<section class="p-strip--light is-deep" id="ceo-video">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Hiring at Canonical</h2>
+      <h3 class="p-heading--2">Mark Shuttleworth, CEO explains the hiring process at Canonical.</h3>
     </div>
-    <div class="row">
-        <h2>Life at Canonical</h2>
+    <div class="col-6 u-embedded-media">
+      <iframe class="u-embedded-media__element" src="https://player.vimeo.com/video/466106710?h=206809c045&amp;autopause=0&amp;player_id=0&amp;app_id=58479" width="560" height="315" frameborder="0" title="What is MicroK8s?"></iframe>
     </div>
-    <div class="row">
-        <ul>
-            <li><a href="/careers/lifestyle">Lifestyle</a></li>
-            <li><a href="/careers/ethics">Ethics</a></li>
-            <li><a href="/careers/travel">Travel</a></li>
-            <li><a href="/careers/progression">Progression</a></li>
-            <li><a href="/careers/diversity">Diversity</a></li>
-        </ul>
+  </div>
+</section>
+<section class="p-strip is-deep">
+{% else %}
+<section class="p-strip--light is-deep">
+{% endif %}
+  <div class="row">
+    <div class="col-3">
+      <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Life at Canonical</h2>
+    </div>
+    <div class="col-9">
+      <ul class="p-list--divided">
+        <li class="p-list__item split-list">
+          <div class="row">
+            <h3 class="col-3 p-heading--2">
+              <a href="/careers/lifestyle">Lifestyle</a>
+            </h3>
+            <p class="col-6">Canonical is amazing for people who are organised, passionate about technology, and like to travel. People at Canonical say the people at Canonical are amazing. That’s special.</p>
+          </div>
+        </li>
+        <li class="p-list__item split-list">
+          <div class="row">
+            <h3 class="col-3 p-heading--2">
+              <a href="/careers/ethics">Ethics</a>
+            </h3>
+            <p class="col-6">We take our responsibility for the most critical elements of every piece of open infrastructure in the software-defined world very seriously, and we are committed to quality in every aspect of our product and services.</p>
+          </div>
+        </li>
+        <li class="p-list__item split-list">
+          <div class="row">
+            <h3 class="col-3 p-heading--2">
+              <a href="/careers/travel">Travel</a>
+            </h3>
+            <p class="col-6">We're uniquely global - we hire the best open source team globally, regardless of nationality or language, creed or colour, and we serve the global market for technology in open infrastructure and open applications.</p>
```

Comment:
The dash here needs wrapping (gh isn't letting me leave the updated line as a suggestion for some reason)